### ISSUE TITLE
Docs: Include design pattern of editing inline

### ISSUE THREAD

**Issue by ErnieAtLYD** [rackerlabs/encore-ui#1638]
As an example in billing:
![image](https://cloud.githubusercontent.com/assets/33945/14897708/334bf7fc-0d51-11e6-9d46-3f0c38ce1e05.png)

**Comment by 100stacks** [comment]
@ErnieAtLYD @glynnis is this something we need to capture in the Framework, or does this need to go through REDD first?

**Comment by ErnieAtLYD** [comment]
I actually got this design pattern _from_ someone at REDD:
https://projects.invisionapp.com/share/HU6WG4WNZ#/screens/139135572

**Comment by glynnis** [comment]
@ErnieAtLYD Can you be more specific with what you want to see documented that isn't already covered by http://rackerlabs.github.io/encore-ui/#/components/rxMetadata? The InVision link you included doesn't really show inline editing, but rather shows a link that opens a modal. 

Just looking for some clarity so I understand what needs to be done before I move this over to the Framework board. Thanks!

**Comment by ErnieAtLYD** [comment]
@glynnis Sure:

- This `(Add)` is for, oh hey, adding. But what about if we need to Delete? Do we use `Edit`? I listed a bunch of options for Billing [EOD-1450](https://jira.rax.io/browse/EOD-1450?focusedCommentId=258568&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-258568).
- "I would honestly expect it to be a pencil icon that creates the modal with the option to delete it altogether." - Alex B.
- "I think historically they’ve always been separate actions rather than combined into a single modal." - Glynnis (and Steven)
- Note that the `(Add)` and `(Edit)` links go to a modal dialogue. Would we make the whole metadata entry a link if it wasn't a modal?

**Comment by glynnis** [comment]
Received an updating from @ErnieAtLYD that this is going to go through a Billing/user-testing process.

@ErnieAtLYD please report back here when you have results, and we can get a story made in JIRA to make whatever changes you require.

**Comment by ErnieAtLYD** [comment]
Update: due to Project Management needing to prioritize urgency, we have had to bypass user testing for now to deliver a solution. This solution was posted at https://invis.io/CM779EY35 .

**Comment by ErnieAtLYD** [comment]
![image](https://cloud.githubusercontent.com/assets/33945/18484976/b56bbcc6-799f-11e6-86c8-8c657e98ffbb.png)

Billing user testing process have been completed although tbh there wasn't really much breakthroughs one way or another. I would make the argument to put this on FRAMEWORK as planned. Also worked on EOD-1657 (internal link) which shows what happens if you don't have the proper permissions.

Additionally, I would suggest to remove @glynnis as she no longer works at Rackspace.

**Comment by ErnieAtLYD** [comment]
also cc-ing: @CITguy as he may be interested with the updated events in this thread.

**Comment by glynnis** [comment]
![](https://media.giphy.com/media/4pMX5rJ4PYAEM/giphy.gif)

**Comment by ErnieAtLYD** [comment]
@glynnis: I mean, unless you _want_ to get Rackspace notifications forever. No complaints from me. ;)

**Comment by glynnis** [comment]
Sorry, not helping 😸  I'm just here to troll
